### ISSUE TITLE
No "New Poll" button visible /  `json_decode` error in the log

### ISSUE THREAD

**Issue by pReya** [nextcloud/polls#3283]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

After upgrading to v28, somehow my Polls app stopped working. Whenever I enter the Polls app, the "New poll" button in the left sidebar is missing. Users can't see any polls.

The admin can see the polls through the "Administration" menu item.

At the same time, every time a user enters the App, there is the following error in the log:

```plaintext
[index] Error: json_decode(): Argument #1 ($json) must be of type string, null given in file '/var/www/html/apps/polls/lib/Db/Preferences.php' line 71
	GET /apps/polls/polls?time=1706647856873
	from xx.xxx.xx.xx by moritz at 30.01.2024, 21:50:56
```

### What did you expect, how polls should behave instead?

- Polls should be visible
- "New Poll" button should be visible

### What steps does it need to replay this bug?

Not sure how to recreate this. Must have happened during the upgrade from 27 to 28.

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

Updated from previous major version (i.e. 3.x.x to 4.x.x)

### Affected polls version

6.0.1

### Which browser did you use, when experiencing the bug?

- [X] Firefox
- [ ] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### NC version

Other/Don't know

### Other Nextcloud version

28.0.1

### PHP engine version

PHP 8.0

### Database engine

MySQL

### Which user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

**Comment by dartcafe** [comment]
See #3216, fixed in #3219 and targeted for version 6.1.0.

As a quick fix the user has to open his polls preferences (lover left corner) once and change a setting.

**Comment by pReya** [comment]
> See #3216, fixed in #3219 and targeted for version 6.1.0.
> 
> As a quick fix the user has to open his Polls preferences (lover left corner) once and change a setting.

Thanks! Sorry for not searching thoroughly.

**Comment by dartcafe** [comment]
> Thanks! Sorry for not searching thoroughly.

relax
